lib/cmap/connection.js

```diff
@@ -2,27 +2,35 @@
 
 const EventEmitter = require('events');
 const MessageStream = require('./message_stream');
-const MongoError = require('../error').MongoError;
-const MongoWriteConcernError = require('../error').MongoWriteConcernError;
-const wp = require('../wireprotocol');
-const apm = require('../connection/apm');
-const updateSessionFromResponse = require('../sessions').updateSessionFromResponse;
-const uuidV4 = require('../utils').uuidV4;
+const MongoError = require('../core/error').MongoError;
+const MongoNetworkError = require('../core/error').MongoNetworkError;
+const MongoWriteConcernError = require('../core/error').MongoWriteConcernError;
+const wp = require('../core/wireprotocol');
+const apm = require('../core/connection/apm');
+const updateSessionFromResponse = require('../core/sessions').updateSessionFromResponse;
+const uuidV4 = require('../core/utils').uuidV4;
 
 const kStream = Symbol('stream');
```

Comment:
Is this the naming convention we're using for constants everywhere? I always tended to prefer uppercase.

Reply:
It's not what we have traditionally used, but I'm taking a cue from [node core](https://github.com/nodejs/node/blob/master/lib/internal/priority_queue.js#L8-L11) and presently only using them for symbols denoting private implementation of a type.

Reply:
Ok fine by me.